src/windows/multiplayer.c

```diff
@@ -264,7 +264,7 @@ static void window_multiplayer_groups_show_group_dropdown(rct_window *w, rct_wid
 
 	for (i = 0; i < network_get_num_groups(); i++) {
 		gDropdownItemsFormat[i] = 2777;
-		gDropdownItemsArgs[i] = (sint64)network_get_group_name(i);
+		gDropdownItemsArgs[i] = (int)network_get_group_name(i);
```

Comment:
I'd actually rather leave it like this for now. It will cause the same error when compiled for 64 bit in future and we can perhaps revisit it when also re-doing the dropdowns/UI.